docs/Example HOWTO.md

```diff
@@ -47,39 +47,39 @@ apt update && apt install bind9
 
 ### Adding the example zone to the DNS server
 
-In this example, we will be working with the domain `gplab.com`.  We must add this domain to the configuration file of the DNS server so that it knows it is responsible for responding to queries for this domain (referred to as a 'zone' in BIND).  Zone information is kept in the following directories, depending upon your Linux version:
+In this example, we will be working with the domain `example.com`.  We must add this domain to the configuration file of the DNS server so that it knows it is responsible for responding to queries for this domain (referred to as a 'zone' in BIND).  Zone information is kept in the following directories, depending upon your Linux version:
 
 CentOS 7 `/etc/bind/named.conf`
 Ubuntu/Raspbian `/etc/bind/named.conf.local`
 
 To add the zone, enter the information below in the appropriate configuration file for your distribution.
 
 ```
-zone "gplab.com" {
+zone "example.com" {
         type master;
-        file "/etc/bind/zones/db.gplab.com";
+        file "/etc/bind/zones/db.example.com";
 };
 ```
 
 ### Configuring the Zone file
 
-You will notice the line `file "/etc/bind/zones/db.gplab.com";` in the configuration above.  This is a pointer to the zone file for the domain `gplab.com`.  In order for the DNS server to work properly, you will need to create this zone file.  But first create the directory `/etc/bind/zones/`.
+You will notice the line `file "/etc/bind/zones/db.example.com";` in the configuration above.  This is a pointer to the zone file for the domain `example.com`.  In order for the DNS server to work properly, you will need to create this zone file.  But first create the directory `/etc/bind/zones/`.
 
 ```
-gp@gplab.com:~ # cd /etc/bind
-gp@gplab.com:~ # mkdir zones
-gp@gplab.com:~ # cd zones
+gp@example.com:~ # cd /etc/bind
```

Comment:
Could we remove gp@example and just use the prompt here and at other places as well?